main/ui/src/main/resources/i18n/strings.properties

```
@@ -142,9 +142,11 @@ preferences.general.startHidden=Hide window when starting Cryptomator
 preferences.general.debugLogging=Enable debug logging
 preferences.general.debugDirectory=Reveal log files
 preferences.general.autoStart=Launch Cryptomator on system start
-preferences.general.pwBackend=Password backend
-preferences.general.pwBackend.gnome=Gnome Keyring
-preferences.general.pwBackend.kde=KWallet
+preferences.general.keychainBackend=Password backend
+preferences.general.keychainBackend.gnome=Gnome Keyring
+preferences.general.keychainBackend.kde=KDE KWallet
+preferences.general.keychainBackend.macSystemKeychain=Mac System Keychain
```

@tobihagemann you decide...

```suggestion
preferences.general.keychainBackend.macSystemKeychain=macOS Keychain Access
```

---

I like the suggested change! 😁👍